<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>access</class>
 <widget class="QWidget" name="access">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>841</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_summary">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>120</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>실시간 출입자 상황</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>오늘 총 입장:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_total_entry">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>32명</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>퇴장:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_total_exit">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>15명</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>50</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>현재 인원:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_current_people">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>50</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>17명</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>날짜 선택:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_date">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>2000-01-01</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="txt_search">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>80</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>검색어 입력...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>출입 유형:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_type">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>80</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>전체</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>입장</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>퇴장</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>출입 구역:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_area">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>80</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>전체</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>창고 A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>창고 B</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>창고 C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>창고 D</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>사무실</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_access">
      <property name="styleSheet">
       <string>QTableWidget {
          border: 1px solid #ddd;
          border-radius: 3px;
        }
        QHeaderView::section {
          background-color: #f2f2f2;
          border: 1px solid #ddd;
          padding: 5px;
        }
       </string>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>70</number>
      </attribute>
      <column>
       <property name="text">
        <string>직원ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>부서</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>입장시간</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>퇴장시간</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>출입구역</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_first">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_prev">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_page">
        <property name="text">
         <string>1 / 5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_next">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_last">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
